packages/mds-stream/kafka/stream-kafka.ts

```diff
@@ -0,0 +1,55 @@
+import { VehicleEvent, Telemetry, Device } from '@mds-core/mds-types'
```

Comment:
Q about this whole file:

We have lots of conditionals like this
```
if (isStreamReady(telemetryStream)) {}
```
Does that mean any events that come in before all the streams have initialized are just swallowed up and silently dropped?

Reply:
Yes, the stream must be explicitly initialized before its used.